engine/runtime/src/main/java/org/enso/interpreter/node/ExpressionNode.java

```diff
@@ -85,6 +90,16 @@ public void setId(UUID id) {
     this.id = id;
   }
 
+  @ExportMessage
+  public boolean hasScope(Frame frame) {
+    throw new UnsupportedOperationException("unimplemented");
```

Comment:
I have created https://www.pivotaltracker.com/story/show/183798184 to track this